README.md

```diff
@@ -64,7 +72,7 @@ At this point, your worker process wil be paused, waiting for the debugger to be
 
 Once the debugger is attached, the process execution will resume and you will be able to debug.
 
-**YOU CAN NOT DEBUG DIRECTLY USING "Start Debugging" IN VISUAL STUDIO DIRECTLY.** You need to use the command line as mentioned in the previous **Run the sample locally** part of this readme.
+**YOU CANNOT DEBUG USING "Start Debugging" IN VISUAL STUDIO DIRECTLY.** You need to use the command line as mentioned in the previous **Run the sample locally** part of this readme.
```

Comment:
I'm a big fan of pointing to the right things with hyperlinks. How about adding a link to that locationhttps://github.com/Azure/azure-functions-dotnet-worker#run-functions-locally?